etc/dnf/automatic.conf

```diff
@@ -5,6 +5,10 @@
 upgrade_type = default
 random_sleep = 0
 
+# Maximum time in seconds to wait until system is on-line and able to
+# connect to remote repositories.
+network_online_timeout = 600
```

Comment:
The default is already set to 600, seems a bit redundant to put the option to the config?

Reply:
I'd like to have this timeout configurable. Of course there is hard-coded default value which would be used when the config file does not contain any value.

Reply:
Yeah, what I meant is since there's the hard-coded default, you don't need to put the option in the default config. The user can add it if they want to override the default. But it's easier for the user to go and change it in the config file without digging for the option in the man page, so lets keep it there.